package/sdbus-cpp/Config.in

```diff
@@ -0,0 +1,18 @@
+config BR2_PACKAGE_SDBUS_CPP
+	bool "sdbus-c++"
+	depends on BR2_INSTALL_LIBSTDCPP
+	depends on BR2_PACKAGE_SYSTEMD
```

Comment:
You may need googletest dependency if you want to enable standalone test.

Reply:
The test suite is disabled in the buildroot package.

Reply:
that's expected and fine